apps/authentication-service/src/resources/clients/clients.test.ts

```diff
@@ -0,0 +1,36 @@
+import { Client } from "auth0";
+import {
+  getClient,
+  createClient,
+  deleteClient,
+} from "@/resources/clients/clientsService";
```

Comment:
I would much prefer a ../clientService vs this file path

Reply:
Totally agree. My IDE auto-added this path. I'll change it.